crates/sui-rpc-benchmark/src/lib.rs

```diff
@@ -51,12 +51,12 @@ pub enum Command {
         endpoint: String,
         #[clap(long, default_value = "50")]
         concurrency: usize,
-        #[clap(long, default_value = "30")]
-        duration_secs: u64,
+        #[clap(long)]
+        duration_secs: Option<u64>,
         #[clap(long, default_value = "requests.jsonl")]
         requests_file: String,
-        #[clap(long, value_delimiter = ',', value_parser = value_parser!(String))]
-        methods_to_skip: HashSet<String>,
+        #[clap(long, value_delimiter = ',')]
+        methods_to_skip: Vec<String>,
```

Comment:
How come this was changed to a `Vec`?

Reply:
the parser `value_parser = value_parser!(String)` does not work and I did not find an easy way to let clap work directly with HashSet. 
Meanwhile I do not follow why HashSet is better than Vec here, at the beginning Vec was passed around and HashSet was constructed multiple times, I got that -- but here Vec is just an intermediate container between CLI string and HashSet, so no extra memory or duplicate codes?